### PR TITLE
Schedule concurrent builds on machines that can scale up

### PR DESCRIPTION
**Pull request by ElenaHenderson**
This PR enables scheduling more than 1 build per machine pipeline using `machine.max_builds` so we can run multiple builds on ec2s machines managed by AWS Buildkite Stacks that can scale up ec2s.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -181,6 +185,7 @@ class Config:
             "supported_filters": ["lang", "name"],
             "offline_warning_enabled": False,
             "publish_benchmark_results": False,
+            "max_builds": 2,
```
Comment (jonkeane) [inline config.py:188]: Any reason not to increase these to even larger?

**Review by jonkeane — APPROVED** [review]
Looks good, my only comment is if we want to increase the cloud instances even more? We pay the same if they are sequential or all at once, yeah?

File: config.py (hunk 1 above)
Comment (ElenaHenderson) [reply config.py:188]: ec2s are scaling to max of 2 right now on infra side. We can increase it here (CI) once we increase max size on infra side.

Comment (jonkeane) [reply config.py:188]: Ok, cool. Thanks!

**Pull request merged**